engineering_tools/VehicleInstallScript.sh

```diff
@@ -31,7 +31,7 @@ echo 'Updating routes folder'
 # if [ ! -d "routes" ]; then # if routes folder does not exist then make it
 #   mkdir /opt/carma/routes
 # fi
-sudo cp -R -r /opt/carma/src/CarmaPlatform/carmajava/route/src/test/resources/routefiles/* /opt/carma/routes/
+sudo cp -R -r /opt/carma/src/CarmaPlatform/carma/route/src/test/resources/routefiles/* /opt/carma/routes/
```

Comment:
You can delete this whole file. It's left over from carma2 and was replaced by the carma script.